lib/icons/trimet.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
```

Comment:
I wonder if there is some kind of way to not have separate icon files and instead read in the icons as needed based on what is in the config.yml file.

Reply:
I'll have a PR for this PR that does this shortly. 

Gonna also try to have a way to customize the itinerary footer.